lib/gooddata/lcm/lcm2.rb

```diff
@@ -311,9 +325,11 @@ def perform(mode, params = {})
             GoodData.logger.info("Running #{action.name} action ...")
 
             # Check if all required parameters were passed
+            params.clear_filters
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)